AnalyticRules/Data_Alert.yaml

```diff
@@ -0,0 +1,28 @@
+id: "1d2c3da7-60ec-40be-9c14-bade6eaf3c49"
+name: "Data Alert"
+description: |
+  'Data Compromised'
+severity: Medium
+status: Available
+requiredDataConnectors: []
+queryFrequency: "5m"
+queryPeriod: "5m"
+triggerOperator: "gt"
+triggerThreshold: 0
+tactics: Impact
+relevantTechniques:
+  - T1578
+  - T1531
+tags:
+  - Commvault
+  - Metallic
+  - Threat Intelligence
+  - Ransomware
+query: |
+    SecurityIncident
+    | where Title has "Cvlt Alert" and Description has "Client" and Description has "Compromised" and Status has "New"
+    | extend extracted_word = extract("Client\\\\s(.*?)\\\\sCompromised", 1, Description)
+    | project TimeGenerated, Title, Description, Status, CustomDetails = extracted_word
+entityMappings: []
+version: "1.0.0.0"
```

Comment:
Again, this applies to all the analytical rules added in this PR.

Please remove the double quotes from `version` and the version should be in format a.b.c, when a is the major version, b is the minor version, and c is a patch.

Please refer to analytical rule shared as example.